clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1607,7 +1604,15 @@ class SyclKernelBodyCreator
   }
 
   bool leaveStruct(const CXXRecordDecl *, FieldDecl *FD) final {
-    const CXXRecordDecl *RD = FD->getType()->getAsCXXRecordDecl();
+
+    QualType FieldTy = FD->getType();
+
```

Comment:
There's some whitespace funnybusiness (perhaps too many new lines?), otherwise this seems OK.

Would prefer 1607 and 1609 go away.

Reply:
Yes. Was literally just correcting that because it annoyed me too :D